bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -12,6 +12,9 @@
 		<label>@text/channel-type.velux.information.label</label>
 		<description>@text/channel-type.velux.information.description</description>
 		<category>NetworkAppliance</category>
+		<tags>
+			<tag>Status</tag>
```

Comment:
Things are moving fast, so i might have missed this rule change, but AFAICT atleast two tags are needed. Property + Point.

Reply:
https://github.com/openhab/openhab-addons/pull/18516#discussion_r2038106383